slo_generator/backends/prometheus.py

```diff
@@ -170,7 +181,12 @@ def count(response):
             return NO_DATA  # no events in timeseries
 
     @staticmethod
-    def _fmt_query(query, window, operators=[], labels={}):
+    def _fmt_query(
+        query: str,
+        window: int,
+        operators: list = [],
```

Comment:
Can we specialize `list` here to indicate which type is in the list?

Reply:
Yes it's a `list[str]` 😄
Done 🚀